rpc/scanner/service.proto

```diff
@@ -23,11 +23,12 @@ message Licenses {
 }
 
 message ScanOptions {
-  repeated string       vuln_type          = 1;
   repeated string       scanners           = 2;
   map<string, Licenses> license_categories = 4;
   bool                  include_dev_deps   = 5;
+  repeated string       pkg_types          = 6;
```

Comment:
You don't have to add a new field when renaming it. Protobuf uses a number for marshaling/unmarshaling. When you change the field type, like int to string, you need a new field.

Reply:
Got it! Updated in https://github.com/aquasecurity/trivy/pull/7104/commits/aeb4f49ffa934ce03dd52c9a74f425cd74cdba59